main.go

```diff
@@ -57,12 +58,22 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
-// WebIdentityTokenPath mount present on operator CSV
-const WebIdentityTokenPath = "/var/run/secrets/openshift/serviceaccount/token"
+const (
+	// WebIdentityTokenPath mount present on operator CSV
+	WebIdentityTokenPath = "/var/run/secrets/openshift/serviceaccount/token"
+
+	// CloudCredentials API constants
+	CloudCredentialGroupVersion = "cloudcredential.openshift.io/v1"
+	CloudCredentialsCRDName     = "credentialsrequests"
 
-// CloudCredentials API constants
-const CloudCredentialGroupVersion = "cloudcredential.openshift.io/v1"
-const CloudCredentialsCRDName = "credentialsrequests"
+	// Pod security admission (PSA) labels
+	pSALabelPrefix = "pod-security.kubernetes.io/"
+	enforceLabel   = pSALabelPrefix + "enforce"
```

Comment:
NIT: pSALabelPrefix <-> PSALabelPrefix

Reply:
that changes the variable behavior, right? (from not exported, to exported). Do we want to export it?

Reply:
all lowercase I think can be problematic: spellcheckers can understand `psa` as a typo. Will change to `PSALabelPrefix` then

Reply:
> that changes the variable behavior, right?

yes

> spellcheckers can understand psa as a typo.

that's not a concern. you can add the text to custom dictionary to avoid this issue.

Reply:
+1 we can always add the text to dictionary

Reply:
done